docs/guide/docs/main/overview.md

```diff
@@ -5,6 +5,19 @@ title: Overview
 
 There are some concepts that you need to understand before you can build a chatbot using Botpress. On this page you will learn the most important ones and how they play together.
 
+## Glossary
+
+**Intent**: It signifies what the user wants to accomplish. Ex: I want to book a flight.
+**Entities**: It's a variable that complements the intent. Ex: I want to book a flight from _Montreal_ to _Paris_ / Cities are entities because they are variable that can be extracted from the intent.
```

Comment:
```suggestion
**Entities**: It's a variable that complements the intent. Ex: I want to book a flight from _Montreal_ to _Paris_ / Cities are entities because they are variables that can be extracted from the intent.
```